src/main/include/subsystems/Elevator.h

```diff
@@ -24,24 +24,44 @@ namespace t34
   class Elevator : public frc2::SubsystemBase
   {
   public:
+
+  enum class WristType
+  {
+    kAlgae,
+    kCoral
+  };
+
     Elevator();
 
     frc2::CommandPtr ElevateToCommand(units::inch_t height);
-    frc2::CommandPtr MoveWristToCommand(units::degree_t angle);
+    frc2::CommandPtr MoveWristToCommand(WristType wrist, units::degree_t angle);
     frc2::CommandPtr MoveUpOnceCommand();
     frc2::CommandPtr MoveDownOnceCommand();
     frc2::CommandPtr MoveToLevelCommand(int level);
+    frc2::CommandPtr MoveToRestCommand();
+
+    frc2::CommandPtr MoveElevatorByPower(double val);
+    frc2::CommandPtr MoveAlgaeWristByPower(double val);
+    frc2::CommandPtr MoveCoralWristByPower(double val);
```

Comment:
Don't forget the naming convention:

```suggestion
    frc2::CommandPtr MoveElevatorByPowerCommand(double val);
    frc2::CommandPtr MoveAlgaeWristByPowerCommand(double val);
    frc2::CommandPtr MoveCoralWristByPowerCommand(double val);
```